src/main/scala/org/broadinstitute/dsde/firecloud/service/UserService.scala

```diff
@@ -54,4 +57,13 @@ class UserService(rawlsDAO: RawlsDAO, userToken: WithAccessToken)(implicit prote
         writableWorkspace = hasWorkspace))
   }
 
+  def getBillingProjectMembership(implicit userToken: WithAccessToken, projectName: String): Future[PerRequestMessage] = {
+    val billingProjectName = RawlsBillingProjectName(projectName)
+    val billingProjectMembership = rawlsDAO.getProjects.map(_.find(_.projectName.equals(billingProjectName)))
```

Comment:
Not sure what the implementation of `rawlsDAO.getProjects` is, so this may not matter - but consider pushing the lookup down a level.